server/http_config_device_handler.go

```diff
@@ -90,21 +89,24 @@ func deviceConfigMap[T any](class templates.Class, dev config.Device[T]) (map[st
 		// from database
 		dc["id"] = configurable.ID()
 
-		props, err := propsToMap(configurable.Properties())
-		if err != nil {
-			return nil, err
-		}
+		if conf.Type == typeTemplate {
```

Comment:
Der Umbau ist nicht sonderlich schön. Aber die `type: custom`-DB Devices sind eine Mischung. Es gibt kein Template. Daher kann `sanitizeMasked` welches die `config` Struktur erzeugt nicht aufgerufen werden. Die Geräte haben aber sowohl eine `id` und können auch `title` und `icon` haben welches wir in der UI zur Differenzierung benötigen.

Diese Lösung funktioniert. Falls dir was eleganteres einfällt, gerne.